proto/osmosis/protorev/v1beta1/genesis.proto

```diff
@@ -25,11 +25,11 @@ message GenesisState {
     (gogoproto.nullable) = false,
     (gogoproto.moretags) = "yaml:\"base_denoms\""
   ];
-  // The pool weights that are being used to calculate the weight (compute cost)
-  // of each route.
-  PoolWeights pool_weights = 4 [
+  // Information that is used to estimate execution time / gas
+  // consumption of a swap on a given pool type.
+  InfoByPoolType info_by_pool_type = 4 [
     (gogoproto.nullable) = false,
-    (gogoproto.moretags) = "yaml:\"pool_weights\""
+    (gogoproto.moretags) = "yaml:\"info_by_pool_type\""
```

Comment:
If we are renaming the proto field, we should depreciate this field and add a new one

Reply:
(Or reserve!)

Reply:
Just to clarify, add a note that this is deprecated and adding a new field?

Reply:
Added back the previous field with a note that it is being deprecated.